react/components/ProductAssemblyOptions/ProductAssemblyOptionItemCustomize.tsx

```diff
@@ -90,13 +90,28 @@ const ModalView: FC<{ closeAction: () => void }> = ({
 
 interface Props {
   buttonProps?: ButtonProps
-  modalProps?: unknown
+  modalProps?: any
 }
 
 interface ButtonProps {
   collapse?: 'left' | 'right' | 'none'
 }
 
+const hasValues = (itemChildren: any) => {
+  const [name] = Object.getOwnPropertyNames(itemChildren)
+  const inputs = itemChildren[name].inputValues
+
+  if (!itemChildren[name]?.valuesOfInputValues) {
+    return false
+  }
+
+  return (
+    inputs.findIndex((input: { label: string }) => {
+      return itemChildren[name].valuesOfInputValues[input.label] !== ''
```

Comment:
How does this behave on numerical inputs?

Reply:
Will work the same way